llama_index/vector_stores/simple.py

```diff
@@ -107,15 +110,45 @@ def __init__(
     def from_persist_dir(
         cls,
         persist_dir: str = DEFAULT_PERSIST_DIR,
+        namespace: Optional[str] = None,
         fs: Optional[fsspec.AbstractFileSystem] = None,
     ) -> "SimpleVectorStore":
         """Load from persist dir."""
+        if namespace:
+            persist_fname = f"{namespace}{NAMESPACE_SEP}{DEFAULT_PERSIST_FNAME}"
+        else:
+            persist_fname = DEFAULT_PERSIST_FNAME
+
         if fs is not None:
-            persist_path = concat_dirs(persist_dir, DEFAULT_PERSIST_FNAME)
+            persist_path = concat_dirs(persist_dir, persist_fname)
         else:
-            persist_path = os.path.join(persist_dir, DEFAULT_PERSIST_FNAME)
+            persist_path = os.path.join(persist_dir, persist_fname)
         return cls.from_persist_path(persist_path, fs=fs)
 
+    @classmethod
+    def from_namespaced_persist_dir(
+        cls,
+        persist_dir: str = DEFAULT_PERSIST_DIR,
+        fs: Optional[fsspec.AbstractFileSystem] = None,
+    ) -> Dict[str, VectorStore]:
+        """Load from namespaced persist dir."""
+        vector_stores: Dict[str, VectorStore] = {}
+        for fname in os.listdir(persist_dir):
```

Comment:
@logan-markewich I think this line may have broken using a remote fs like gcsfs?